### PR TITLE
Create additional proof design and functionality

### PR DESCRIPTION
**Pull request by T-Damer**
- Works with `Work email` proof

### CODE REVIEW

File: src/components/ProofsCard.tsx [hunk 1]
```diff
@@ -84,6 +84,18 @@ function Proofs() {
                 <ListOfAvailableZKProofs />
               </>
             )}
+            <ProofSection
+              title={
+                <>
+                  Additional proofs{' '}
```
Comment (T-Damer) [inline src/components/ProofsCard.tsx:90]: I guess it's `<BodyText bold />`

File: src/icons/StaticArrow.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
+export default function ({ turnUp }: { turnUp?: boolean }) {
```
Comment (T-Damer) [inline src/icons/StaticArrow.tsx:1]: You can do `<Arrow disabled />` and for open-closed state use `<Arrow disabled turnDown />`

File: src/icons/Email.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+export default function () {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="14"
```
Comment (T-Damer) [inline src/icons/Email.tsx:5]: Better remove them so the `svg` will rescale properly and use `div` wrapper

File: src/hooks/useEmailForm.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import { useEffect, useState } from 'react'
+
+export default function () {
+  const [email, setEmail] = useState('')
+  const [emailIsValid, setEmailIsValid] = useState(false)
+
+  const isEmailValid = (email: string) => {
+    const re =
```
Comment (T-Damer) [inline src/hooks/useEmailForm.ts:8]: It's better to use `<input type="email" />`, it validates `email` automatically

File: src/components/WorkProof.tsx [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+  function onSendSecret(secret?: string) {
+    if (secret) {
+      console.log(secret)
+    }
+  }
+
+  return (
+    <ProofLine className={proofLineContainer}>
+      <div className={workTitleContainer}>
+        <span>{`Work email ${domain ? `@${domain}` : ''}`}</span>
```
Comment (T-Damer) [inline src/components/WorkProof.tsx:67]: Pretty complicated check, it's better to go with 
```html
        <span>Work email {domain ? `@${domain}` : ''}</span>
```
Maybe you can go without `$` and ```` , I am not sure

File: src/components/WorkProof.tsx [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+  function onSendEmail(email?: string) {
+    if (email) {
+      const [_, domain] = email.split('@')
+      setDomain(domain)
+    }
+  }
+
+  function onSendSecret(secret?: string) {
+    if (secret) {
```
Comment (T-Damer) [inline src/components/WorkProof.tsx:59]: Better go without curly braces

**Comment by backmeupplz** [comment]
Considering all the changes happening now, this PR will be outdated quite soon. Re-create this PR when SC is stable

**Pull request closed without merging**